modules/system/classes/UpdateManager.php

```diff
@@ -533,11 +533,14 @@ public function updatePlugin($name)
     }
 
     /**
-     * Removes an existing plugin
+     * Rollback an existing plugin
+     *
      * @param string $name Plugin name.
+     * @param null $stopOnVersion If this parameter is specified, the process stops after
```

Comment:
Instead of `null`, this param should be a `string`.
```suggestion
     * @param string $stopOnVersion If this parameter is specified, the process stops after
```